source/doc-pages/guides/basics/mappers.md

```diff
@@ -99,11 +99,14 @@ users.map_with(:entity).to_a
 Mappers can also be used to convert tuples that are returned from ROM [commands](commands).
 
 ```ruby
-create_user = ROM.env.command(:users).create
-create_user.call id: 3, name: 'jack', email: 'jack@doo.org'
+ROM.env.command(:users).create.call(
+  id: 3, name: 'jack', email: 'jack@doo.org'
+)
```

Comment:
Small thing, but I don't like showing examples with global `ROM.env` so I at least assign a specific command to a local var. We probably need to have some convention for all guides, all ROM components should be injected as dependencies, using `ROM.env.*` is a smell.